packages/visx-grid/src/grids/GridPolar.tsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
+import cx from 'classnames';
+import { Group } from '@visx/group';
+import GridAngle from './GridAngle';
+import GridRadial from './GridRadial';
+
+import { CommonGridProps, GridScale } from '../types';
+
+export type GridPolarProps<Scale extends GridScale> = CommonGridProps & {
+  /**
+   * If specified, the arc of each radial grid line will have this thickness, useful for fills.
+   */
+  arcThickness?: number;
+  /**
+   * The class name applied to the angle grid group.
+   */
+  classNameAngle?: string;
+  /**
+   * The class name applied to all radial lines.
+   */
+  classNameRadial?: string;
+  /**
+   * The end angle of the arc of radial grid lines.
+   */
+  endAngle?: number;
+  /**
+   * The color applied to the fill of the radial lines.
+   */
+  fillRadial?: string;
+  /**
+   * Radius which determines the start position of polar lines
+   */
+  innerRadius?: number;
+  /**
+   * Classname applied to angle line path.
+   */
+  lineClassNameAngle?: string;
+  /**
+   * Classname applied to radial line path.
+   */
+  lineClassNameRadial?: string;
+  /**
+   * Style object set as the angle line path style attribute.
+   */
+  lineStyleAngle?: object;
+  /**
+   * Style object set as the radius line path style attribute.
+   */
+  lineStyleRadial?: object;
```

Comment:
Same for this one re specificity

```suggestion
  lineStyleRadial?: LineProps & Omit<React.SVGProps<SVGLineElement>, keyof LineProps>;
```